app/models/event.rb

```diff
@@ -0,0 +1,4 @@
+class Event < ActiveRecord::Base
+	has_many :users
```

Comment:
as Event and User are connected by UserEvent model, you can't just connect these two model as you did. Check this article http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association out for know about through association.